build_and_test.cmd

```diff
@@ -0,0 +1,5 @@
+dotnet restore --configuration Debug src
```

Comment:
I think all of this can be reduced to a single line `dotnet test src`. since 

 * test does a build and restore OOTB
 * Debug is the default
 * all projects will be built

Reply:
True. I'm trying to mimic appveyor.yml, maybe that should be changed?